flappyBird/sketch.js

```diff
@@ -62,7 +62,7 @@ function setup() {
 
   pauseBecauseDead = false;
 
-  population = new Population(1000);
+  population = new Population(10);
```

Comment:
Why did you need to change that value?

Reply:
I changed this value because too big population can cause lags. Of course you can change population to 50 or 100 and it will be even better.